packages/ui/src/proposals/components/AddProposalButton.tsx

```diff
@@ -34,4 +35,21 @@ export const AddProposalButton = () => {
       Add new proposal
     </TransactionButton>
   )
+
+  if (!api) return <Tooltip tooltipText="Connecting to api">{txButton()}</Tooltip>
+
+  if (!areProposalSlotsAvailable)
+    return (
+      <Tooltip
+        tooltipTitle="MAX_ACTIVE_PROPOSALS"
```

Comment:
Wondering should we rename the title to  something like this?
```suggestion
        tooltipTitle="Max active proposals limit reached"
```